.github/workflows/bullmq-tests.yml

```diff
@@ -0,0 +1,47 @@
+name: bullmq-tests
```

Comment:
So is this not run automatically anymore?

Reply:
they are currently disabled anyways